src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -140,9 +140,66 @@ commands going across the wire, communicating with the browser.
 
 ## Using programmatically (Node) {: #node }
 
-### Launching Chrome {: #nodelaunch }
+### The Puppeteer API {: #puppeteer }
 
-In the previous section, we [started Chrome manually](#cli) using
+[Puppeteer](https://github.com/GoogleChrome/puppeteer) is a Node library
+developed by the Chrome team. It provides a high-level API to control headless
+(or full) Chrome. It's similar to other automated testing libraries like Phantom
+and NightmareJS, but it only works with the latest versions of Chrome.
+
+Among other things, Puppeteer can be used to easily screenshots, create PDFs,
+navigate pages, and fetch information about those pages. I recommend the library
+if you want to get up and running quickly with automated browser testing.
```

Comment:
Change

'if you want to get up and running quickly with automated browser testing'

to

'if you want to quickly automate browser testing'